### PR TITLE
[DOC] Zeppelin 0.6.0 release note

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Add Zeppelin 0.6.0 release note and change latest version of Zeppelin

### What type of PR is it?
Documentation

### Todos

* [x] Wait for https://issues.apache.org/jira/browse/INFRA-12205 to be resolved 
   * [x] Check download mirror link is availble
   * [x] Check KEY file link is valid

### How should this be tested?
Outline the steps to test the PR here.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: releases/zeppelin-release-0.6.0.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+-->
+{% include JB/setup %}
+
+## Apache Zeppelin Release 0.6.0
+
+The Apache Zeppelin community is pleased to announce the availability of the 0.6.0 release.
+
+The community put significant effort into improving Apache Zeppelin since the last release, focusing on having new backend support, implementing authentication and authorization for enterprise.
+More than 70+ contributors provided 360+ patches for new features, improvements and bug fixes.
+More than 200+ issues has been resolved.
```
Comment (AhyoungRyu) [inline releases/zeppelin-release-0.6.0.md]: `has`-> `have` would be correct i think :)

File: releases/zeppelin-release-0.6.0.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+   * Interpreter/Confidential/Configuration Navigation Menu Moved to Right Top [Dropdown Menu](../../docs/0.6.0/quickstart/explorezeppelinui.html#settings)
+
+<br />
+You can visit [issue tracker](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12332761&styleName=Html&projectId=12316221) for full list of issues being resolved.
+
+
+<br />
+### Contributors
+
+This release would not be possible without the following community members's contributions:
```
Comment (AhyoungRyu) [inline releases/zeppelin-release-0.6.0.md]: How about `members's` -> `members'` ?

**Comment by Leemoonsoo** [comment]
Looks good to me

**Comment by prabhjyotsingh** [comment]
LGTM!

**Comment by anthonycorbacho** [comment]
This [issue](https://issues.apache.org/jira/browse/ZEPPELIN-1104) is missing from the release note

**Comment by khalidhuseynov** [comment]
LGTM! #880 was missing the issue, so i fixed it under https://issues.apache.org/jira/browse/ZEPPELIN-1111. I believe it's new feature as well, and could be included in release notes. except that everything looks good!

File: releases/zeppelin-release-0.6.0.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Several interpreters are merged into JDBC interpreter. If you were using one of below interpreters, you can continue to use it by setting additional properties and dependencies in JDBC interpreter. Examples of JDBC interprer setting can be found [here](../../docs/0.6.0/interpreter/jdbc.html#examples).
+
+   * Hive
+   * Pheonix
+   * Tajo
+
+### New features
+   * [Shiro Authentication](../../docs/0.6.0/security/shiroauthentication.html)
+   * [Notebook Authorization](../../docs/0.6.0/security/notebook_authorization.html)
+   * [Azure](..//docs/0.6.0/storage/storage.html#notebook-storage-in-azure), [ZeppelinHub](../../docs/0.6.0/storage/storage.html#storage-in-zeppelinhub) Notebook Storage
```
Comment (minahlee) [inline releases/zeppelin-release-0.6.0.md]: @khalidhuseynov ZeppelinHub storage feature is stated in this line :)

Comment (khalidhuseynov) [inline releases/zeppelin-release-0.6.0.md]: i must have missed that! thanks! :)

File: releases/zeppelin-release-0.6.0.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+This release includes new backend support for
+
+   * [Alluxio](../../docs/0.6.0/interpreter/alluxio.html)
+   * [Hbase](../../docs/0.6.0/interpreter/hbase.html)
+   * [HDFS](../../docs/0.6.0/interpreter/hdfs.html)
+   * [JDBC](../../docs/0.6.0/interpreter/jdbc.html)
+   * [Livy](../../docs/0.6.0/interpreter/livy.html)
+   * [Python](../../docs/0.6.0/interpreter/python.html)
+   * [SparkR](../../docs/0.6.0/interpreter/r.html)
```
Comment (felixcheung) [inline releases/zeppelin-release-0.6.0.md]: just "R"?

**Comment by felixcheung** [comment]
LGTM

File: download.md [hunk 5]
```diff
@@ -21,39 +21,61 @@ limitations under the License.
...
+    ([pgp](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz.asc),
+     [md5](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz.md5),
+     [sha](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz.sha))
 
 
 
 ### Verify the integrity of the files
 
-It is essential that you [verify](https://www.apache.org/info/verification.html) the integrity of the downloaded files using the PGP or MD5 signatures. This signature should be matched against the [KEYS](https://www.apache.org/dist/incubator/zeppelin/KEYS) file.
+It is essential that you [verify](https://www.apache.org/info/verification.html) the integrity of the downloaded files using the PGP or MD5 signatures. This signature should be matched against the [KEYS](https://www.apache.org/dist/zeppelin/KEYS) file.
```
Comment (bzz) [inline download.md]: Regarding the project KEYS file - https://www.apache.org/dist/zeppelin/KEYS returns

```
Not Found

The requested URL /dist/zeppelin/KEYS was not found on this server.
```

Is there a way for us to:
 -  transfer out KYES file from [incubator](https://www.apache.org/dist/incubator/zeppelin/KEYS)
 - add Mina's key there
?

Comment (felixcheung) [inline download.md]: hmm, perhaps have incubator releases point to  https://www.apache.org/dist/incubator/zeppelin/KEYS

then have the 0.6.0 release to point to https://www.apache.org/dist/zeppelin/KEYS
I assume before we release 0.6.0 we need to have https://www.apache.org/dist/zeppelin/KEYS

Comment (minahlee) [inline download.md]: @bzz Sorry, I missed mentioning about KEYS file link. I just updated description.

Actually I already added my key and located it under https://dist.apache.org/repos/dist/release/zeppelin/KEYS

As far as I understand from [Apache release process doc](http://www.apache.org/dev/release-publishing.html#distribution_dist), all contents of https://dist.apache.org/repos/dist/release/zeppelin will be pushed to https://www.apache.org/dist/zeppelin once svnpubsub set.

So I believe [INFRA-12205](https://issues.apache.org/jira/browse/INFRA-12205) will also solve this issue. Let me know if there is any further action I should take :)

File: releases/zeppelin-release-0.6.0.md (hunk 4 above)
Comment (minahlee) [inline releases/zeppelin-release-0.6.0.md]: The reason I wrote as `SparkR` here is that R interpreter belongs to spark group. I believe Zeppelin will have R interpreter under r group one day, like the way that general python interpreter is added recently apart from pyspark interpreter. What do you think?

**Comment by minahlee** [comment]
Merge if there is no more discussion

**Pull request closed without merging**